pkg/tfbridge/info.go

```diff
@@ -672,6 +672,13 @@ type GolangInfo struct {
 	// InternalDependencies are blank imports that are emitted in the SDK so that `go mod tidy` does not remove the
 	// associated module dependencies from the SDK's go.mod.
 	InternalDependencies []string
+
+	// Specifies how to handle generating a variant of the SDK that uses generics.
+	// Allowed values are the following:
+	// - "none": do not generate a generics variant of the SDK
+	// - "side-by-side": generate a side-by-side generics variant of the SDK under the x subdirectory
+	// - "only-generics": generate a generics variant of the SDK only
```

Comment:
I would love this comment to explain what `""` is interpreted as. 

Otherwise LGTM

Reply:
Updated the comment. `"none"` is currently the default when not specified. Although, at some point in the near future we intend to flip the default to `"side-by-side"` so all providers get a generic SDK by default.